### DISCUSSION TITLE
Lychee 6.1.0 zip not updated

### DISCUSSION THREAD

**Discussion by prbt2016** [LycheeOrg/Lychee#2737] [Q&A] [answered]
Hello @ildyria ,

I was trying to download Lychee 6.1.0 (latest) version from https://github.com/LycheeOrg/Lychee/releases/download/v6.1.0/Lychee.zip

However, when I checked the zip , version.md shows 6.0.1. Is it a packaging issue?.

Could you please check and upload the latest zip here https://github.com/LycheeOrg/Lychee/releases/download/v6.1.0/Lychee.zip

**Comment by ildyria** [comment, marked as answer]
Sorry. My mistake, indeed packaging issue.
This should be fixed. I updated the zip file.

**Reply by prbt2016** [reply]
Hello @ildyria ,

Thanks :+1:
